knowledge_base/_Glossary.md

```diff
@@ -0,0 +1,250 @@
+# Glossary
+
+As of 240305, this starter glossary is in-progress and needs improvement. If you notice unclear wording or conceptual confusions, please open a PR tethered to the #4800 general documentation ticket!
+
+Glossary terms referenced in a given term's definition are designated with CAPITAL letters. Relevant models and related entries may also be referenced; see [Subscription](#subscription) for an example.
+
+## Contents
+
+- [Chain Concepts](#chain-concepts)
+  * [Chain Entity](#chain-entity)
+  * [Chain Node](#chain-node)
+  * [Delegate Contract](#delegate-contract)
+  * [Module](#module)
+- [Common App](#common-app)
+  * [Bridge](#bridge)
+  * [Comment](#comment)
+  * [Community](#community)
+  * [Core](#core)
+  * [Forum](#forum)
+  * [Group](#group)
+  * [Link](#link)
+  * [Member](#member)
+  * [Notification](#notification)
+  * [Page Scope](#page-scope)
+  * [Profile](#profile)
+  * [Reaction](#reaction)
+  * [Poll](#poll)
+  * [Role](#role)
+  * [Sessions](#sessions)
+  * [Sign-In](#sign-in)
+  * [Stake](#stake)
+  * [Subscription](#subscription)
+  * [Template](#template)
+  * [Thread](#thread)
+  * [Topic](#topic)
+  * [User](#user)
+  * [Wallet](#wallet)
+  * [Votes](#votes)
+- [Hexagonal Architecture](#hexagonal-architecture)
+  * [Port](#port)
+  * [Adapter](#adapter)
+- [Praxis](#praxis)
+  * [Blocked](#blocked)
+  * [Fast-Follow](#fast-follow)
+  * [Issues, Tickets, & Stories](#issues-tickets--stories)
+- [Change Log](#change-log)
+
+## Chain Concepts
+
+### Chain Base
+
+An architectural layer of blockchain atop which NETWORK protocols are built.
+
+### Chain Entity
+
+A Chain Entity is a backend object, stored in the ChainEntities table, which consists of chain event bundles and values received from a smart contract.
+
+### Chain Node
+
+An endpoint, referring to external RPC servers, used to fetch updates and query data from blockchains.
+
+### Delegate Contract
+
+A delegate contract is a smart contract that contains the functions, events, and all business logic for a proxy contract.
+
+### Module
+
+Some CHAINS use modules rather than contracts. Modules are used predominantly within the Cosmos and Substrate ecosystems.
+
+### Network
+
+An ecosystem of CHAIN NODES, or blockchain, built atop a CHAIN BASE.
```

Comment:
Network as we use it in the codebase is not ecosystem but some mixture of ecosystem and specific chains. Semantically it basically refers to the database field alone, which is not standardized or having a real referent besides "which client code to load".